integration-tests/integration/common/common.spec.js

```diff
@@ -373,6 +394,18 @@ And('I should see the success banner message {string}', $string => {
   cy.get('.moj-banner--success').contains($string).should('exist')
 })
 
+And('I should see a textarea with id {string}', $comment => {
+  cy.get(`#${$comment}`).should('exist')
```

Comment:
Possibly rename comment -> id?

Reply:
Sorry, this is needed.. its a lift & shift from case-summary spec.